yogstation/code/modules/antagonists/traitor/datum_mindslave.dm

```diff
@@ -11,4 +11,19 @@
 	name = "Mindslave Objective"
 	martyr_compatible = TRUE
 	completed = TRUE
-	explanation_text = "Serve your master no matter what!"
\ No newline at end of file
+	explanation_text = "Serve your master no matter what!"
+
+/datum/antagonist/greytide
+	name = "Greytider"
+	antagpanel_category = "Mindslave"
+	job_rank = ROLE_TRAITOR
+	show_in_antagpanel = FALSE
+	antag_moodlet = /datum/mood_event/focused
+	can_hijack = FALSE
+	var/mob/living/carbon/master // I think I need to keep this for log purposes
+
+/datum/objective/greytide
+	name = "Mindslave Objective" // alt "Take what you want, the station is yours. Avoid needless harm you are still a crewmemeber."
+	martyr_compatible = TRUE
+	completed = TRUE
+	explanation_text = "Break things, take what you want, avoid harming anyone doing so. Remain loyal to your master and assistant brothers."
```

Comment:
You define it here but define it later when generating the objective

Reply:
Don't I need to keep the explanation text in the implant so I can add the masters real name to the  text. And keep it in the mindslave datum so its defined.